airbyte-integrations/bases/bases-destination-jdbc/build.gradle

```diff
@@ -29,7 +29,4 @@ dependencies {
 
     integrationTestJavaImplementation project(':airbyte-integrations:bases:standard-destination-test')
     integrationTestJavaImplementation libs.connectors.testcontainers.postgresql
-
-    implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
-    integrationTestJavaImplementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)
```

Comment:
I mentioned this in our 1:1, but I’m excited to see these drop because they complicated the Java CDK build process. Happy to see them retiring.